Products/Zuul/routers/manufacturers.py

```diff
@@ -42,8 +42,11 @@ def removeProducts(self, products):
         """
         facade = self._getFacade()
         facade.removeProducts(products)
-        audit('UI.Manufacturers.RemoveProducts', products=products)        
-        return DirectResponse.succeed()
+        audit('UI.Manufacturers.RemoveProducts', products=products)
+        return DirectResponse.succeed(
+            "Job to remove product(s) and update product class for "
+            "instances has been scheduled."
```

Comment:
This can be more succinct:

> Scheduled job to remove %s product(s).

where `%s` is the number of products the user selected for removal.